src/frontend/src/App.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
+
+import 'fomantic-ui-css/semantic.css';
+import { Container, Header } from 'semantic-ui-react';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
+        <Container> <Header className="yellow"> Athenaeum</Header></Container>
```

Comment:
Small nit, let's format this so that way one component per line; or is this Prettier's recommendation?

Reply:
Ah, Good point...if anything we should add this into prettier to make sure it finds it.